src/oData/jsonToOData.ts

```diff
@@ -241,7 +241,5 @@ export const orderbyBuilder = <T>(
 	if (refScheme) {
 		fieldPath += `/${refScheme.replace(/\[(.*?)\]/g, '').replace(/\./g, '/')}`;
 	}
-	return {
-		[fieldPath]: direction,
-	};
+	return [`${fieldPath} ${direction}`, `id ${direction}`];
```

Comment:
Btw this (and probably the above as well) could also be:
```suggestion
	return [{ [fieldPath]: direction }, { id: direction }];
```